### ISSUE TITLE
WhateverGreen i5-9600K (UHD630): BS animations are not smooth, the awake process leads to green background -> lost scaling + freezes

### ISSUE THREAD

**Issue by GoodBoris** [acidanthera/bugtracker#1350]
Hello everyone. I had an issue with **iGPU only** configuration from the very beginning of Hackintosh experience.
My configuration is:
Processor: Intel Core i5-9600K (UHD630)
Motherboard: Asus Z390-I Gaming
Monitor: Samsung **4K** U28E590
Connectors type: HDMI/DP
DVMT preallocated: 128Mb

When I tried to awake my PC everything started working except turning on the monitor. When I pressed the monitor's power button manually the system was crashing and rebooting. I spent a lot of time trying to patch USB connectors but this did not lead to a successful result. A few releases ago of whatevergreen the issue was partly auto fixed. Instead of crashing, when the monitor's power button is manually pressed the system started to show me a green background-color ![IMAGE 2020-12-06 01:08:37](https://user-images.githubusercontent.com/6509965/101260063-9966bd00-375f-11eb-9c57-b720e5d64c1a.jpg) for ~2-second -> flashed -> Log In page with lost resolution/scaling and 30GHz feelings (all graphical operations start to work very slowly).

First of all, I want to say, that I read up and down all the instructions for patching the framebuffers. I tried about ~100 combinations of different settings including platform-id/device-id changing, injecting fake IGPU, patching connectors, changing SMBIOS, and so on. The best result I achieved is just increasing VRAM to 2000 leads to a properly awaking process via DP connector. HDMI usage still shows me a green background first and the power button must be pressed manually.
The only platform that works for me 'somehow' is '0x3E9B0007'. With any other, the system can not be booted. By the way, that macMini 8,1 platform is predefined automatically by the library if I don't apply _AAPL,ig-platform-id_ explicitly.

You could somehow live with the problems described above. But after the system is updated to BS, I got a new permanent issue. Almost all the system animations are not smooth anymore. It feels like the iGPU just can't handle the system graphics, which is hard for me to believe.

I assume that both described above issues are related to each other and we have a bug in the kext extension. I tried to solve the issue with guys from tonymac forum helping. Nobody can help me. I think we tried whatever is possible (including different bootloaders). I created a thread in tonymac with my screen capturing and screenshots from Hackintool and VideoProc software: https://www.tonymacx86.com/threads/after-big-sur-install-mac-os-graphic-doesnt-smooth.307403

I hope that somebody from the dev team can help me. Because delays in the system graphic are very annoying and do not allow to work normally. Let me know if any logs can help you and let me know how to grub the, please.

Thanks in advance!

**Comment by Andrey1970AppleLife** [comment]
We don't support Hackintool.

**Comment by GoodBoris** [comment]
> We don't support Hackintool.
Hi Andrey. Thanks for the reply. But I don't understand why the ticket was closed. My issue isn't about Hackintool supporting. I tried to tell that my iGPU doesn't work properly. Hackintool was mentioned to show my tries and steps about fixes my current state. And all tries are failed.
I'm pretty sure something wrong in whatevergreen extension.
Let me know if something not understandable.
Thank you.
